_posts/2024-04-15-heapdump-oome-native.adoc

```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title: 'Obtaining heap dump on OutOfMemoryError with Quarkus native'
+date: 2024-04-15
```

Comment:
Let's target a publication on Tuesday? We can merge tomorrow and get it published automatically on Tuesday and tweet about it on Tuesday? Friday is a slow day for blog posts.

Note that if you want it published tomorrow, I'm around!

Reply:
Tuesday is fine, I'll ammend it.